app/models/sample.rb

```diff
@@ -80,4 +83,15 @@ def empty_entity?
   def has_entity_id?
     entity_ids.compact.any?
   end
+
+  def transfer_to(new_owner)
```

Comment:
Perhaps the name could reflect the in-transition state, for example `transferring_to()`? The `transfer_to` feels like it's being done right now, which is no longer the case.

Reply:
Maybe `#start_transfer`?